### PR TITLE
refresh slim after adding terms for OBA

### PR DESCRIPTION
**Pull request by ar-ibrahim**
- added CHEBI:76172 (loxoprofen) for https://github.com/EBISPOT/efo/issues/2195

### CODE REVIEW

File: chebi_slim.obo
Comment (rays22) [inline chebi_slim.obo:1]: This update is not needed for this PR.

File: chebi_slim.owl
Comment (rays22) [inline chebi_slim.owl:1]: This update is not needed for this PR.

**Review by rays22 — APPROVED** [review]
In my reading, the instructions in the [README file](https://github.com/obophenotype/chebi_obo_slim) say that you
1. add new terms in the first step (first PR), and
2. create a release in a subsequent step (in a separate PR).

I expect to see changes only in the `seeed.txt` file. You should run the `sh odk.sh make all` to generate new `.obo` and `.owl` files for a new release and include them in a separate PR. Ontologies that import the `chebi_obo_slim` rely on the files in the public release, so the `.obo` and `.owl` files generated in the current PR are not used for automated ODK imports.
I will approve this PR, but going forward, please follow the instructions in the README.

**Comment by ar-ibrahim** [comment]
> In my reading, the instructions in the [README file](https://github.com/obophenotype/chebi_obo_slim) say that you
> 
> 1. add new terms in the first step (first PR), and
> 2. create a release in a subsequent step (in a separate PR).
> 
> I expect to see changes only in the `seeed.txt` file. You should run the `sh odk.sh make all` to generate new `.obo` and `.owl` files for a new release and include them in a separate PR. Ontologies that import the `chebi_obo_slim` rely on the files in the public release, so the `.obo` and `.owl` files generated in the current PR are not used for automated ODK imports. I will approve this PR, but going forward, please follow the instructions in the README.

OK good to know thanks for that Ray. I guess we can also update the [OBA editors guide](https://github.com/obophenotype/bio-attribute-ontology/blob/master/docs/editors-guide.md) accordingly with a link to the chebi slim README file.

**Pull request merged**